dev/generate-cuda.py

```diff
@@ -26,10 +26,7 @@
     "awkward_content_reduce_zeroparents_64",
     "awkward_ListOffsetArray_reduce_global_startstop_64",
     "awkward_IndexedArray_reduce_next_fix_offsets_64",
-    "awkward_Index8_to_Index64",
-    "awkward_IndexU8_to_Index64",
-    "awkward_Index32_to_Index64",
-    "awkward_IndexU32_to_Index64",
+    "awkward_Index_to_Index64",
     "awkward_carry_arange",
```

Comment:
This is an improvement. I think those separate kernels were defined before I got into the cadence of making such things specializations of a single kernel.